javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,183 @@
+//Item array removal
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = names.indexOf("Ahmad");
+
+  if(nameToRemove>-1){
+      names.splice(nameToRemove,1); //remove one item only
+  };
+  
+  console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+  //When will we be there? exircise 
+
+
+  const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+
+  function calculateTime(speed, km){
+const timeInMinutes =(km/speed)*60; // turn the num to minutes
+const hours = Math.floor(timeInMinutes/60); //gets the hours
+const rhours= Math.floor(hours);  // should give us the hours
+const minutes =Math.floor(hours-rhours)*60; //takes the decimal part and multiplate it by 60
+const rminutes= Math.floor(minutes); // should give us the minutes
+console.log (`${rhours} hours and ${rminutes} minutes`);
+return;
+    
+    };  
+
+
+calculateTime(travelInformation.destinationDistance, travelInformation.speed);
+ //not sure why the output is always 0 I tried many ways to solve it..
+  
```

Comment:
Nice with the comments.
remember your function calculateTime take speed as the first variable and km as the second variable.
The function should call accordingly. like calculateTime( travelInformation.speed, travelInformation.destinationDistance).
I don't think you need to change const timeInMinutes =(km/speed)*60 b/c you changed back to hours format in const hours.
you only need to use Math.floor on rhours not on hours, because Math. floor gives you the closest integer value. 
(hours-rhours )return 0 value. That is why your output is 0.

Reply:
thank you so much Haile for your comments they helped me a lot